### PR TITLE
Show messages from the verification process in notification bar instead of modal dialogue

### PR DESCRIPTION
**Pull request by c633**
This attempts to fix both #73 and #30

### CODE REVIEW

**Comment by arlolra** [comment]
Thanks! I'll take a look as soon as possible.

**Comment by arlolra** [comment]
Please rebase this branch after today's merge.

**Comment by c633** [comment]
@arlolra: rebased.

File: chrome/content/ui.js [hunk 1]
```diff
@@ -522,6 +616,11 @@ var ui = {
       break;
     case "contact-added":
       ui.onContactAdded(aObject);
```
Comment (arlolra) [inline chrome/content/ui.js]: break;

**Comment by arlolra** [comment]
I merged this in 43c2de62 with some cleanup in 07fe3400.  Mostly just code deduplication, with some minor fixes when cancelling.

Great work, Huy!

**Comment by arlolra** [comment]
If you have any thoughts on #76, let me know.

**Pull request closed without merging**